packages/devextreme-cli/testing/linting.js

```diff
@@ -0,0 +1,61 @@
+const reactEnv = require('./env.react');
+const vueEnv = require('./env.vue');
+const angularEnv = require('./env.angular');
+const minimist = require('minimist');
+const fs = require('fs');
+const ESLint = require('eslint').ESLint;
+
+const args = minimist(process.argv.slice(), {
+    default: {
+        template: 'all'
+    },
+    alias: {
+        t: 'template'
+    }
+});
+
+const envs = {
+    'react': reactEnv,
+    'angular': angularEnv,
+    'vue': vueEnv
+};
+
+async function lint(env) {
+    console.log('START LINT: ' + env.engine);
+    let eslint = new ESLint({
+        useEslintrc: false,
+        overrideConfigFile: `./testing/lint-config/${env.engine}.eslintrc`,
+        ignore: false
+    });
+    let report = await eslint.lintFiles([
+        `./testing/sandbox/${env.engine}/my-app/src/**/*.${env.engine === 'angular' ? 'ts' : 'js'}`
```

Comment:
Maybe it's better to store the information about file's extension right in env files?

Reply:
`eslint.lintFiles` requires a string or array of string. Tried adding the file pattern to the config file. as a result, skips linting.

Reply:
corrected to a shorter entry